packages/web/config/ibc-overrides.ts

```diff
@@ -33,7 +33,7 @@ type AdditionalData = Partial<
 const TestnetIBCAdditionalData: Partial<
```

Comment:
Do we still need this file? All of these data should be on the new asset list

Reply:
I think it perhaps could be removed, but I'd like to get to removing that when we do the new bridge project that fully leverages the bridge providers. I'm hoping with that project we can delete a lot of this legacy code and config.

Reply:
Sounds good. In the meantime, how about we move the deposit/withdraw override URLs since they change too frequently (need to open PRs) and are low risk to refactor?

Reply:
I will adopt those with my work on new asset page. I'll include them in the table response.